src/main/java/com/faforever/client/domain/CoopResultBean.java

```diff
@@ -18,6 +18,7 @@
 public class CoopResultBean {
   @EqualsAndHashCode.Include
   ObjectProperty<Integer> id = new SimpleObjectProperty<>();
+  // TODO: Why returns `null` value?
```

Comment:
Probably is just old data that doesn't have playerNames

Reply:
Can I remove it?

Reply:
is it used anywhere?

Reply:
Nope